Runtime/Scripts/MobiledgeXLogger.cs

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2018-2021 MobiledgeX, Inc. All rights and licenses reserved.
+ * MobiledgeX, Inc. 156 2nd Street #408, San Francisco, CA 94105
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using UnityEngine;
+
+namespace MobiledgeX
+{
+    public class MobiledgeXLogger
```

Comment:
Maybe "Logger", since developer already chose the MobiledgeX namespace via Using to shorten the object name, but it's currently near guaranteed there's no collisions named this way.

Reply:
Good point